xml/art_raspberry-pi.xml

```diff
@@ -288,15 +198,14 @@
    <sect3 xml:id="sec-rpi-platform-software-yast">
     <title>&yast;</title>
     <para>
-     &yast; is the installation and configuration framework for &sle;. It is
-     popular for its ease of use, flexible graphical interfaces and the capability
-     to customize your system quickly during and after installation. &yast;
-     can be used to configure your entire system: You can configure hardware,
-     set up networking, manage system services, and tune your security settings.
-     All these tasks can be accessed from the &yast; control center. To start
-     it, choose &yast; in the menu or run the command <command>xdg-su -c
-     yast2</command>. You will be prompted to enter the password of the
-     &rootuser; user.
+     &yast; is a user-friendly installation and configuration framework for
```

Comment:
I suggest "tool" rather than the possibly-nebulous term "framework". Tool is after all what the T in YAST stands for.